python/cudf/cudf/core/frame.py

```diff
@@ -2403,6 +2403,19 @@ def _copy_type_metadata(
 
         return self
 
+    def _copy_interval_data(self, other, include_index=True):
+        for name, col, other_col in zip(
+            self._data.keys(), self._data.values(), other._data.values()
+        ):
+            if isinstance(other_col, cudf.core.column.IntervalColumn):
+                self._data[name] = cudf.core.column.IntervalColumn(col)
+                # col.as_interval_column()
```

Comment:
Dead code?

Reply:
I feel like I added this in for a reason :sweat_smile: I can't remember what though. I'll take it out and check that it doesn't break anything

Reply:
Whoops, sorry, realised you meant the comment. Removing it :D